site/content/_index.md

```diff
@@ -1,5 +1,25 @@
 # Code Review During Development
 
+From small data analysis scripts to large simulation software, **quality
+software is essential to quality research**. Code readability,
+maintainability and extensibility are key to research outputs that
+can be reproduced and built upon.
+
+But how can you assess the readability of your code? Have someone else
+read it!  Whether your are an advanced programmer or a beginner, code
+reviews will greatly improve the quality of your software by giving
+you access to human feedback on your code and programming
+practices. Code reviews are also opportunities for **knowledge
+transfer**. Whether you participate as a reviewer or reviewee, code
+reviews are opportunities to learn from your colleagues .. or teach
+them a few tricks!
+
+**This website aims at providing a clear, opinionated workflow
```

Comment:
Maybe use a different word from "opinionated"? IME this usually has a somewhat negative connotation

"well-informed"?

to convey sth like "our aim is to integrate latest findings"